tests/neg/i16954.scala

```diff
@@ -0,0 +1,5 @@
+class Test:
+  def test =
+    type y
```

Comment:
You can use a `class` instead of a `type` to define something for which `classOf` should be valid.